metadata_backend/api/middlewares.py

```diff
@@ -73,15 +73,15 @@ async def jwt_authentication(req: Request, handler: Callable) -> Response:
 
         # JWK and JWTClaims parameters for decoding
         key = environ.get('PUBLIC_KEY', None)
-        # TODO: get Oauth2 public key
-        # TODO: verify audience claim
+
+        # Get Oauth2 public key and verify audience claim here
```

Comment:
I'd delete these "here should be something" -lines totally. If you want to mark the lines where something should happen, link lines to issue #96

Reply:
Fair enough :D